miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -31,6 +31,13 @@ abstract class MiniApp internal constructor() {
         versionId: String
     ): MiniAppDisplay
 
+    /**
+     * Fetches meta data information of a mini app.
+     * @return [MiniAppInfo] for the provided appId of a mini app
+     * @throws [MiniAppSdkException] when fetching fails from the BE server for any reason.
+     */
+    abstract suspend fun fetchMiniAppInfo(appId: String): MiniAppInfo
```

Comment:
Maybe this can be `fetchInfo`? `fetchMiniAppInfo` sounds redundant since the class is already named `MiniApp`.

Reply:
☯️